lhotse/cut.py

```diff
@@ -467,7 +467,10 @@ def __add__(self, other: 'CutSet') -> 'CutSet':
         return CutSet(cuts={**self.cuts, **other.cuts})
 
 
-def make_cuts_from_features(feature_set: FeatureSet) -> CutSet:
+def make_cuts_from_features(
```

Comment:
There is another function just below called `make_cuts_from_supervisions` - was it not sufficient or not visible? Feedback on this is welcome, I'm still trying to figure out what is the best way to make different cutting strategies discoverable.

Reply:
Sorry, I didn't notice it. The function `make_cuts_from_supervisions` is sufficient.